### PR TITLE
PUD-1447: Use RecallLite for the recallList to improve performance.

### PR DESCRIPTION
**Pull request by philsegal**

### CODE REVIEW

File: server/@types/manage-recalls-api/models/RecallLite.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/* istanbul ignore file */
+/* tslint:disable */
+/* eslint-disable */
+
+import {RecallResponse} from "./RecallResponse";
+
+export type RecallLite = {
```
Comment (dazoakley) [inline server/@types/manage-recalls-api/models/RecallLite.ts:7]: Not `RecallResponseLite`?

**Review by dazoakley — DISMISSED** [review]
Looks ok, but I'd question the model name.

File: server/@types/manage-recalls-api/models/RecallLite.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    assigneeUserName?: string;
+    createdByUserId: string;
+    createdDateTime: string;
+    dossierEmailSentDate?: string;
+    dossierTargetDate?: string;
+    firstName: string;
+    inCustodyAtAssessment?: boolean;
+    inCustodyAtBooking?: boolean;
+    lastName: string;
+    lastUpdatedDateTime: string;
```
Comment (jonnywyatt) [inline server/@types/manage-recalls-api/models/RecallLite.ts:17]: Optional property?

File: server/@types/manage-recalls-api/models/RecallLite.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    licenceNameCategory: RecallResponse.licenceNameCategory;
```
Comment (jonnywyatt) [inline server/@types/manage-recalls-api/models/RecallLite.ts:18]: Optional property?

**Review by jonnywyatt — CHANGES_REQUESTED** [review]

File: server/@types/manage-recalls-api/models/RecallLite.ts (hunk 1 above)
Comment (philsegal) [reply server/@types/manage-recalls-api/models/RecallLite.ts:7]: Good shout

**Review by jonnywyatt — APPROVED** [review]

File: server/@types/manage-recalls-api/models/RecallLite.ts (hunk 2 above)
Comment (philsegal) [reply server/@types/manage-recalls-api/models/RecallLite.ts]: as discussed, not option as sent as same value as createdDateTime initially

File: server/@types/manage-recalls-api/models/RecallLite.ts (hunk 3 above)
Comment (philsegal) [reply server/@types/manage-recalls-api/models/RecallLite.ts]: set as FIRST_LAST on create (for now at least)

**Pull request merged**